### PR TITLE
chore: api examples to es6

### PR DESCRIPTION
**Pull request by jajugoguma**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
- Change examples in api document(jsdoc) and README to ES6
- Prettier table of contents in README

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: README.md [hunk 1]
```diff
@@ -11,17 +11,33 @@
...
-* [Examples](#-examples)
-* [Install](#-install)
-* [Usage](#-usage)
-* [Browser Support](#-browser-support)
-* [Pull Request Steps](#-pull-request-steps)
-* [Contributing](#-contributing)
-* [TOAST UI Family](#-toast-ui-family)
-* [License](#-license)
+- [TOAST UI Component : Color Picker](#toast-ui-component--color-picker)
+  - [🚩 Table of Contents](#-table-of-contents)
```
Comment (jungeun-cho) [inline README.md:15]: 목차 위에 영역은 굳이 작성하지 않아도될것같아서요.

File: src/js/factory.js [hunk 2]
```diff
@@ -35,9 +35,11 @@ function generateId() {
  *  @param {string} [options.detailTxt='Detail'] - text for detail button.
  *  @param {boolean} [options.usageStatistics=true] - Let us know the hostname. If you don't want to send the hostname, please set to false.
  * @example
- * var colorPicker = tui.colorPicker; // or require('tui-color-picker')
+ * import colorPicker from 'tui-color-picker'; // ES6
+ * // const colorPicker = require('tui-color-picker'); // CommonJS
+ * // const colorPicker = tui.colorPicker;
```
Comment (jungeun-cho) [inline src/js/factory.js:40]: 마지막꺼는 브라우저 환경이라고 코멘트 추가하면 좋을 것 같아요.

**Review by jungeun-cho — APPROVED** [review]
리뷰완료합니다.

**Pull request merged**